src/views/evaluation/evaluations.js

```diff
@@ -44,8 +52,95 @@ function EvaluationsTable(options = {}) {
         <col style="width:30%">
         <col style="width:10%">
     </colgroup>
-        ${EvaluationsTableHeader(options.headings)}
-        ${EvaluationTableBody(options.rows)}
+        ${EvaluationsTableHeader(tableHeaders.headings)}
+        ${EvaluationTableBody(tableData)}
 </table>
     `
 }
+
+
+const detailsCreator = function (el) {
```

Comment:
I find that these method names aren't generic enough or sufficiently indicative of their purpose. What exactly are we trying to create here? Any sort of details? Details for what? Is this going to be an object? A factory function?